backend/lcfs/web/api/organization/views.py

```diff
@@ -5,28 +5,39 @@
 from sqlalchemy.ext.asyncio import AsyncSession
 from sqlalchemy.exc import SQLAlchemyError
 from sqlalchemy import select
+from starlette import status
+from sqlalchemy.orm import selectinload
 from starlette.responses import Response
 
 from lcfs.db import dependencies
 from lcfs.db.models import UserProfile
 from lcfs.db.models.Organization import Organization
 from lcfs.db.models.OrganizationAddress import OrganizationAddress
 from lcfs.db.models.OrganizationAttorneyAddress import OrganizationAttorneyAddress
-from lcfs.web.api.organization.schema import OrganizationSchema, OrganizationCreateSchema, OrganizationUpdateSchema, OrganizationUserSchema, OrganizationSummarySchema
+from lcfs.web.api.organization.schema import (
+    OrganizationSchema,
+    OrganizationCreateSchema,
+    OrganizationUpdateSchema,
+    OrganizationUserSchema,
+    GetOrganizationResponse,
+)
 from lcfs.web.core.decorators import roles_required
 
 logger = getLogger("organization")
 router = APIRouter()
 get_async_db = dependencies.get_async_db_session
 
+
 # TODO: Implement permission check for this route to ensure that
 # only authorized users can access it.
-@router.post("/", response_model=OrganizationSchema, status_code=status.HTTP_201_CREATED)
+@router.post(
+    "/", response_model=OrganizationSchema, status_code=status.HTTP_201_CREATED
+)
 @roles_required("Government", "Adminstrator")
 async def create_organization(
```

Comment:
I'm getting a duplicate key error when trying to create a new organization. Are you able to create in your local environment?

Reply:
I've been using the seeded data to test so I havnt run in to this issue. Ill test it out on my local

Reply:
this should be fixed now. will need to delete and seed the db again. the sequence was off when seeding so it caused dupe keys